hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestStorageContainerManager.java

```diff
@@ -289,8 +291,7 @@ public void testBlockDeletionTransactions() throws Exception {
           return false;
         }
       }, 1000, 10000);
-      Assert.assertTrue(helper.getAllBlocks(containerIDs).isEmpty());
-
+      Assert.assertTrue(helper.verifyBlocksWithTxnTable(containerBlocks));
```

Comment:
As of now @errose28, TestStorageContainerManager uses SCHEMA version 2 so we don't need separate functions for schema 1 and schema 2. But yes in follow-up Jira we can make changes in TestStorageContainerManager so that it runs on both the schema versions and then we can add functions schema version-wise.

Reply:
Thanks, @errose28 for the suggestion, right now code will only use SCHEMA version 2. Right now I don't think we should use SCHEMA_LATEST to check schema versions as it will never use version 1. In follow-up Jira we will make changes as we did in TestBlockDeletingService so that it will run on both the version and then we can check the SCHEMA version as per your suggestion.